stop.sh

```
@@ -0,0 +1,11 @@
+#!/bin/bash
+CURRENT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+source "$CURRENT_DIR/.env"
+source "$COMPOSE_HOME/.env"
+
+cd "$CURRENT_DIR/$COMPOSE_HOME"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)